api/src/main/java/org/openmrs/module/sync/server/ServerConnection.java

```diff
@@ -250,6 +252,11 @@ private static ConnectionResponse httpPostTransmit(String url, String username,
 
 			method.setRequestEntity(new MultipartRequestEntity(parts, method.getParams()));
 
+			// Support authentication header
+			String authString = username + ":" + password;
+			String authHeader = "Basic " + Base64.encodeBase64String(authString.getBytes(StandardCharsets.UTF_8));
+			method.setRequestHeader("Authorization", authHeader);
+

```

Comment:
Not sure what's up with all of the whitespace changes in this PR, but the above 4 lines are all that is added.